src/components/ServiceList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import FilterList from "../components/FilterList";
+import useServicesList from "../hooks/useServicesList";
+import { Card, CardContent } from "@baltimorecounty/dotgov-components";
+
+const ServiceList = () => {
+  const serviceItems = useServicesList();
+
+  const serviceCard = ({ name, department, url, icon, rank }) => (
+    <Card id={name}>
+      <CardContent>
+        <h3>{name}</h3>
+        <p>{department}</p>
+        <a href={url}>{name}</a>
+      </CardContent>
+    </Card>
+  );
+
+  return <FilterList items={serviceItems} renderItem={serviceCard} />;
```

Comment:
this is looking good, but well want to render an item with the bootstrap grid classes. If i'm remembering correctly the grid should be 4 wide on desktop, and then well address after we see how that responds.

Reply:
done